modules/tealium_adobevisitor/android/src/main/kotlin/com/tealium/flutter/modules/adobevisitor/TealiumAdobeVisitorPlugin.kt

```diff
@@ -181,6 +183,24 @@ class TealiumAdobeVisitorPlugin : FlutterPlugin, MethodChannel.MethodCallHandler
         }
     }
 
+    fun getUrlParameters(call: MethodCall, result: MethodChannel.Result) {
```

Comment:
The function definition for `getUrlParameters` in the Dart file doesn't have any parameters. You shouldn't don't need the `call` parameter here since you're not sending any arguments with the function call.